solr/core/src/java/org/apache/solr/cloud/api/collections/ReindexCollectionCmd.java

```diff
@@ -138,7 +138,7 @@ public static State get(Object p) {
       return states.get(p);
     }
 
-    static Map<String, State> states =
+    static final Map<String, State> states =
         Collections.unmodifiableMap(
             Stream.of(State.values())
                 .collect(Collectors.toMap(State::toLower, Function.identity())));
```

Comment:
`Collectors.toUnmodifiableMap` will simplify this a little as well.

Reply:
addressed in c324ae6e07fb6134708d9e67d38172ec91c4f27d